Dockerfile

```diff
@@ -21,22 +24,11 @@ ARG CPU=native
 RUN apt-get update && \
     apt-get -y install --no-install-recommends \
         ca-certificates \
-        libelf-dev
+        libelf-dev \
+        libbpf0
```

Comment:
as discussed pls check compatibility with other versions of linux . UPF work as on bare metal, and most of the people prefer to test on bare metal for testing/comparing performances.

if required these changes can be put under ubuntu 20 specific check ( if version <-> ubuntu 20.x) and for 'else' part we can keep older code.

Reply:
Aren't we using ubuntu focal version for BESS ? So it should be Ubuntu 20.04 LTS . So adding libbpf0 looks reasonable. For ubuntu focal this is libbpf 0.5.0 version . [https://packages.ubuntu.com/focal-updates/libbpf-dev](https://packages.ubuntu.com/focal-updates/libbpf-dev
)